lib/Repository.js

```diff
@@ -445,6 +448,33 @@ class Repository extends Requestable {
    }
 
    /**
+    * Add collaborator to this repo.
+    * @see https://developer.github.com/v3/repos/collaborators/#add-user-as-a-collaborator
+    * @param {Object} options - options with username as collaborator - permission (optional)
+    * @param {Requestable.callback} [cb] - will receive the list of PRs
```

Comment:
@nmoya every single method has this param in the Repository class. They get past to the `_request` method. So I'm not sure what the reason is.